.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# IDEs
+.idea
```

Comment:
This really should be in the user's gitignore.

Reply:
Done